hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/CryptoPreTransactionHandlerImpl.java

```diff
@@ -44,12 +44,12 @@ public CryptoPreTransactionHandlerImpl(@Nonnull final AccountStore accountStore)
 
     @Override
     /** {@inheritDoc} */
-    public TransactionMetadata preHandleCryptoCreate(final TransactionBody tx) {
-        final var op = tx.getCryptoCreateAccount();
+    public TransactionMetadata preHandleCryptoCreate(final TransactionBody txn) {
+        final var op = txn.getCryptoCreateAccount();
```

Comment:
Missing null check for txn

Reply:
I also recommend adding @Nonnull annotations